src/BusyIcon/index.tsx

```diff
@@ -0,0 +1,66 @@
+import type { FC } from "react";
+
+import "./index.scss";
+import clsx from "clsx";
+
+export const BusyIconSizes = ["sm", "base", "lg"] as const;
```

Comment:
Prefer `camelCase` for values.

```suggestion
export const busyIconSizes = ["sm", "base", "lg"] as const;
```